hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMContainerMetrics.java

```diff
@@ -84,6 +84,14 @@ public void getMetrics(MetricsCollector collector, boolean all) {
             stateCount.get(DELETING.toString()))
         .addGauge(Interns.info("DeletedContainers",
             "Number of containers in deleted state"),
+            stateCount.get(DELETED.toString()))
+        .addGauge(Interns.info("TotalContainers",
+            "Number of all containers"),
+          stateCount.get(OPEN.toString())+
+            stateCount.get(CLOSING.toString())+
+            stateCount.get(QUASI_CLOSED.toString())+
+            stateCount.get(CLOSED.toString())+
+            stateCount.get(DELETING.toString())+
```

Comment:
Nit: Can this be converted to a loop over all states summing individual values?  That way if a new state is introduced, no further change in counting totals will be required.

Reply:
Thanks @adoroszlai  for the suggestion,I tried to modify it.